src/middlewared/middlewared/plugins/alert.py

```diff
@@ -53,8 +53,8 @@ class AlertSourceRunFailedAlertClass(AlertClass):
 class AlertSourceRunFailedOnBackupNodeAlertClass(AlertClass):
     category = AlertCategory.SYSTEM
     level = AlertLevel.CRITICAL
-    title = "Alert Check Failed (Backup Node)"
-    text = "Failed to check for alert %(source_name)s on backup node:\n%(traceback)s"
+    title = "Alert Check Failed (Backup Controller)"
```

Comment:
The word is that we will also call it Passive Controller, not Backup

Reply:
Is that right, @wblock ?

Reply:
@freenas/docs-team ?

Reply:
Unfortunately, no.  After the UI meeting, I brought this up to the Product Management meeting, and was told that Active/Passive is wrong, and we have always been Active/Standby.  I am supposed to research this, but that will have to be a separate set of tickets.